frontend/src/app/assess/[projectId]/fairness-bias/dataset-testing/report/components/DatasetSnapshot.tsx

```
@@ -0,0 +1,74 @@
+            {preview.rows.length > PAGE_SIZE && (
+                <div className="flex items-center justify-between border-t border-slate-100 dark:border-gray-800 pt-4" data-html2canvas-ignore="true">
+                    <p className="text-sm text-slate-500">
+                        Showing {(currentPage - 1) * PAGE_SIZE + 1} to {Math.min(currentPage * PAGE_SIZE, preview.rows.length)} of {preview.rows.length} rows
+                    </p>
+                    <div className="flex items-center gap-2">
+                        <button
+                            onClick={() => setCurrentPage(p => Math.max(1, p - 1))}
+                            disabled={currentPage === 1}
+                            className="p-2 rounded-lg hover:bg-slate-100 dark:hover:bg-gray-800 disabled:opacity-50 disabled:cursor-not-allowed transition-colors"
+                        >
+                            <ChevronLeft className="w-5 h-5 text-slate-600 dark:text-slate-400" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)